content/docs/started/local/_index.md

```diff
@@ -0,0 +1,48 @@
++++
+title = "Local Installation"
+description = "Get started with Kubeflow on local hardware"
+weight = 1
++++
+
+   * [MiniKF](/docs/started/local/getting-started-minikf/)
+      * MiniKF is a fast and easy way to get started with Kubeflow.
+      * It installs with just two commands and then you are up for
+	      experimentation, and for running complete Kubeflow Pipelines.
+      * MiniKF runs on all major operating systems (Linux, macOS, Windows).
+
+   * [Minikube](/docs/started/local/getting-started-minikube/)
+      * Minikube uses virtualization applications like 
+        [VirtualBox](https://www.virtualbox.org/) or [VMware
+        Fusion](https://www.vmware.com/products/fusion.html) to host the VM
+	      and provides a CLI that you can use outside the VM.
+      * Minikube defines a fully-baked
+       [ISO image](https://en.wikipedia.org/wiki/ISO_image) that contains a
+        minimal operating system and Kubernetes already installed.
+      * This option may be useful if you are just starting to learn and already
+	      have one of the virtualization applications installed.
+
```

Comment:
microk8s will work on any linux system, local or in the cloud.  Addtionally, with the advent of WSL2 on Windows, it'll work on Windows too. In the context of local, this will work without a virtual machine. I can make this edit afterwards or as part of this submission.  Preference?

Reply:
After discussing this with Sarah, I'm closing this PR for now. Please give your input on how best to restructure the Getting Started guide here: https://github.com/kubeflow/website/issues/889